packages/button/src/ButtonContent/ButtonContent.tsx

```diff
@@ -50,6 +50,17 @@ export const ButtonContent = (props: ButtonContentProps) => {
     return unregisterRipple;
   }, [rippleRef, variant, darkMode, disabled, theme]);
 
+  const spinner =
+    loadingIndicator &&
+    React.cloneElement(loadingIndicator, {
+      className: cx({
+        [centeredSpinnerStyles]: !loadingText,
+      }),
+      sizeOverride: buttonSpinnerSize[size],
+      colorOverride: spinnerColor[theme],
+      ['data-testid']: 'lg-button-spinner',
```

Comment:
Should spread other props passed to Spinner

Reply:
updated